sqlx-postgres/src/row.rs

```diff
@@ -48,3 +50,23 @@ impl ColumnIndex<PgRow> for &'_ str {
             .map(|v| *v)
     }
 }
+
+impl Debug for PgRow {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "PgRow: ")?;
+
+        let mut debug_map = f.debug_map();
+        for (index, column) in self.columns().iter().enumerate() {
+            if let Ok(value) = self.try_get_raw(index) {
+                debug_map.entry(
+                    &column.name,
+                    &Postgres::fmt_value_debug(&<PgValueRef as ValueRef>::to_owned(&value)),
+                );
+            } else {
+                debug_map.entry(&column.name, &"ERROR");
```

Comment:
I'd make the string `(decode error)` just to make it clear the value is not literally the string `"ERROR"`.

Reply:
Changed